uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-actions.component.js

```diff
@@ -178,6 +181,20 @@ define([
     }
 
     /**
+     * Opens the leave request popup
+     *
+     * @param {Object} event
+     * @param {Object} leaveRequest
+     * @param {String} leaveType
+     * @param {String} selectedContactId
+     * @param {Boolean} isSelfRecord
+     */
+    function openLeavePopup (event, leaveRequest, leaveType, selectedContactId, isSelfRecord) {
+      event.stopPropagation();
```

Comment:
1. It was copied from HTML
2. It is needed because the `leave-request-actions.component` sits inside `leave-request-dropdown-actions.component`. And both have on click listeners. So when the child is clicked the parent's click event should not be called.